src/mscorlib/shared/System/SpanHelpers.Char.cs

```diff
@@ -167,6 +168,91 @@ public static unsafe int IndexOf(ref char searchSpace, char value, int length)
             }
         }
 
+        public static unsafe int LastIndexOf(ref char firstChar, int startIndex, char value, int length)
+        {
+            Debug.Assert(length >= 0);
+            
+            fixed (char* pChars = &firstChar)
+            {
+                char* pCh = pChars + startIndex;
+                char* pEndCh = pCh - length;
+
+#if !netstandard11
+                if (Vector.IsHardwareAccelerated && length >= Vector<ushort>.Count * 2)
+                {
+                    const int elementsPerByte = sizeof(ushort) / sizeof(byte);
+                    length = (((int)pCh & (Unsafe.SizeOf<Vector<ushort>>() - 1)) / elementsPerByte) + 1;
```

Comment:
Doesn't this need to be `Vector<byte>`?  I think either you don't need `elementsPerByte`, or you should use `Vector<byte>`.

Reply:
Aren't `Vector<byte>.Count` and `Unsafe.SizeOf<Vector<ushort>>()` equivalent in value? In either case, we would still need `elementsPerByte`, no?

What about IndexOf: https://github.com/dotnet/coreclr/blob/master/src/mscorlib/shared/System/SpanHelpers.Char.cs#L96?

Reply:
You're correct.  I was thinking `Vector<T>.Count`, not `Unsafe.SizeOf`.  Please ignore.